### PR TITLE
Update REST API docs for legacy features no longer supported by new implementation

### PR DESCRIPTION
**Pull request by antonecma**
@jekh would you like to clean up nonused REST API parameters from code and documentation (described here #510, #510 )?

### CODE REVIEW

File: browsermob-rest/src/main/java/net/lightbody/bmp/proxy/bricks/ProxyResource.java [hunk 1]
```diff
@@ -478,28 +478,6 @@ public void process(BrowserMobHttpRequest request, Har har) {
             } catch (NumberFormatException e) {
             }
         }
-        String payloadPercentage = request.param("payloadPercentage");
```
Comment (jekh) [inline browsermob-rest/src/main/java/net/lightbody/bmp/proxy/bricks/ProxyResource.java]: This functionality still works when using the legacy implementation, so we should keep this code in, even though the new implementation doesn't support it. Optionally, you could check to see if proxy is an instanceof BrowserMobProxyServer and log a warning that the feature is not supported with the new implementation.

**Comment by antonecma** [comment]
Sorry, but my Java skill level is lower than zero. Can you explain how can I check out if the legacy implementation is running, please?

**Comment by jekh** [comment]
You have to do an instanceof check. I'd suggest just not removing that code.

**Comment by jekh** [comment]
No update so I'm going to go ahead and close this.

**Pull request closed without merging**